openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/TaskTab.component.tsx

```diff
@@ -266,7 +267,9 @@ export const TaskTab = ({
         rest.onAfterClose?.();
         rest.onUpdateEntityDetails?.();
       })
-      .catch((err: AxiosError) => showErrorToast(err));
+      .catch((err: AxiosError) =>
+        showErrorToast(getErrorText(err, t('server.unexpected-error')))
```

Comment:
Hello @Ashish8689, any reason we are using `getErrorText` here directly?

Reply:
yes, since the status code was coming `FORBIDDEN =`403` for not having access, and we are blocking it in the UI 
 for certain places to display. That's we have gone for this!